src/PatientChecking/PatientChecking/Controllers/PatientController.cs

```diff
@@ -132,5 +141,89 @@ public IActionResult Detail(int patientId)
 
             return View(model);
         }
+
+        [HttpPost]
+        public IActionResult Update(PatientDetailViewModel model)
+        {
+            if (!ModelState.IsValid)
```

Comment:
Why do we have `ModelState.IsValid` here but I don't see any Validation annotation in `PatientDetailViewModel`?

Reply:
This has been removed. Please review this @vietvo361